src/lineage/templates/conflict_template.md

```diff
@@ -41,7 +41,9 @@ that you must resolve before merging this pull request!
 
     ```console
     git add {{#conflict_file_list}}{{.}} {{/conflict_file_list}}
-    git commit --message="Resolve Lineage conflicts."
+    # You may append to the default merge commit message that git creates for
```

Comment:
As someone who copies and pastes the block of commands, I'd recommend moving the directive out of the comments here and probably either deleting it entirely or putting it in the text above like:

> **Note:** Feel free to append to the default merge commit message, but _do not_ overwrite it, as the content it provides is useful.

Reply:
I'm happy to move it, but the comments shouldn't cause you any difficulties if you copy the whole block wholesale.  Is this a `zsh` thang?

Reply:
Addressed in commit 21a3634.

Reply:
You're right, it wouldn't cause actual technical issues - more of an aesthetic issue I guess.